src/main/scala/com/linagora/openpaas/gatling/calendar/utils/AttendeeUtils.scala

```diff
@@ -3,19 +3,9 @@ package com.linagora.openpaas.gatling.calendar.utils
 import com.linagora.openpaas.gatling.utils.RandomUuidGenerator.randomUuidString
 
 object AttendeeUtils {
-  def generateRandomAttendeesInICalFormat(numberOfAttendees: Int = 100): String = {
-    var attendees = "";
+  def generateRandomAttendeesInICalFormat(numberOfAttendees: Int = 100): String =
+    (1 to numberOfAttendees).map(index => constructAttendee(index)).mkString(",\n")
 
-    1 to numberOfAttendees foreach { index => {
-      val attendeeEmail = s"attendee-$randomUuidString@fake.org"
-
-      attendees = attendees.concat(
-        s"""["attendee",{"partstat": "NEEDS-ACTION","rsvp": "TRUE","role": "REQ-PARTICIPANT","cutype": "INDIVIDUAL"},"cal-address","mailto:$attendeeEmail"]""".stripMargin
-      )
-
-      if (index != numberOfAttendees) attendees = attendees.concat(",\n")
-    }}
-
-    attendees;
-  }
+  private def constructAttendee(index: Int): String =
+      s"""["attendee",{"partstat": "NEEDS-ACTION","rsvp": "TRUE","role": "REQ-PARTICIPANT","cutype": "INDIVIDUAL"},"cal-address","mailto:attendee-$index-$randomUuidString@fake.org"]""".stripMargin
```

Comment:
randomUuidString is not a session key so it will always be the same value.
Is it a problem or is it what you want ?

Reply:
It doesn't really matter here as I already have `index` to differentiate the attendees.